usaspending_api/search/models/transaction_search.py

```diff
@@ -319,6 +319,7 @@ class TransactionSearch(models.Model):
     native_american_owned_busi = models.BooleanField(null=True)
     native_hawaiian_owned_busi = models.BooleanField(null=True)
     native_hawaiian_servicing = models.BooleanField(null=True)
+    initial_report_date = models.TextField(null=True)
```

Comment:
Two pieces of feedback here:
1. Why is this in the  `# Exclusively FPDS` set of columns? I think the `# Dates` section would make a lot more sense. Can you move it there?
2. This should be a `DateTimeField` instead of a text field.